### ISSUE TITLE
Don't display map type icons in Mac (not displayed in Win + too cluttered)

### ISSUE THREAD

**Issue by GoogleCodeExporter** [lixun910/geoda#205]
```
Mac: Don't display icons for map types like box map, percentile, SD map etc. 
(too cluttered + rate icons also currently not displayed. They're already not 
displayed in Windows, so this way we'll have consistency.
```

Original issue reported on code.google.com by `jkoschin...@gmail.com` on 27 Jul 2015 at 12:34

**Comment by lixun910** [comment]
fix is applied in 1.7.49

**Comment by jkoschinsky** [comment]
Verified in GeoDa 1.7.63
